core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/PerUserEntitlementManager.java

```diff
@@ -64,7 +64,7 @@ private static EntitlementManager load(BrooklynProperties properties, String typ
     public PerUserEntitlementManager(BrooklynProperties properties) {
```

Comment:
I've seen this manager replaces the deprecated `PerUserEntitlementManagerWithDefault` - shall we remove that one now that we are almost at 0.10.0-SNAPSHOT? 
not sure what's the rule of thumb for that :)